examples/console/stats.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"sync/atomic"
+	"time"
+)
+
+type stats struct {
+	paymentsSent             int64
+	paymentsReceived         int64
+	bufferedPaymentsSent     int64
+	bufferedPaymentsReceived int64
+}
+
+func (s *stats) Reset() {
+	atomic.StoreInt64(&s.paymentsSent, 0)
```

Comment:
❓ jw why you chose atmoic here (since I haven't used before). A quick google search says it's because `atomic` is quicker but `mutex` is better for complex data structures, is that why you chose atomic here?